readme.md

```diff
@@ -249,7 +249,8 @@ Default: `'text'`
 The parsing method. Can be `'text'`, `'json'` or `'buffer'`.
 
 The promise also has `.text()`, `.json()` and `.buffer()` methods which return another Got promise for the parsed body.\
-It's like setting the options to `{responseType, resolveBodyOnly: true}` but without affecting the main Got promise.\
+It's like setting the options to `{responseType: true, resolveBodyOnly: true}` but without affecting the main Got promise.
```

Comment:
The `responseType` is not a boolean. It's a string. I think this change should be reverted.

Reply:
Yeah, not sure what I was doing there. 🤷‍♂️ https://github.com/sindresorhus/got/pull/1064/commits/fe1c6a8f2ea87a841d45e294081fe336405dbea2